examples/with-sfcc/README.md

```diff
@@ -0,0 +1,27 @@
+# Example Next.js app with Salesforce Commerce Cloud
+
+This example shows how to create a headless ecommerce application using Next.js, [Salesforce commerce cloud](https://www.salesforce.com/products/commerce-cloud/overview/), and [Tailwind](https://tailwindcss.com).
+
+## Deploy your own
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or view the demo [here](https://salesforce-cloud-commerce.vercel.app/)
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/with-sfcc&project-name=with-sfcc&repository-name=with-sfcc&env=SFDC_CLIENT_ID,SFDC_SECRET,SFDC_ORGANIZATIONID,SFDC_SHORTCODE,SFDC_SITEID&envDescription=API%20Keys%20from%20SFCC%20needed%20to%20run%20this%20application.)
+
+Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+
+## How to use
+
+Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/with-sfcc) with [npm](https://docs.npmjs.com/cli/init), [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/), or [pnpm](https://pnpm.io) to bootstrap the example::
+
+```bash
+npx create-next-app --example with-sfcc nextjs-sfcc-app
+# or
+yarn create next-app --example with-sfcc nextjs-sfcc-app
+# or
+pnpm create next-app --example with-sfcc nextjs-sfcc-app
+```
```

Comment:
Should look like this  (cannot suggest this because the code fences also use three "`")

https://github.com/vercel/next.js/blob/562b5a380ead4a41abdb1574e262a8a24bcbe0a3/examples/hello-world/README.md?plain=1#L15-L25